randovania/games/am2r/logic_database/Hydro Station.txt

```diff
@@ -357,8 +357,10 @@ Extra - minimap_data: [{'x': 34, 'y': 18}]
               # Item requirements
               Can Use Bombs
               Morph Ball and Power Bombs ≥ 3
-          # Energy requirements
-          Combat (Intermediate) or Normal Damage ≥ 130
+          Any of the following:
+              # Energy requirements
+              Combat (Intermediate) or Normal Damage ≥ 200
```

Comment:
honestly, wonder if hitless arachnus should be bumped up. dodging the green bouncing fireball, as well as the blue claw swipes can be a bit of a pain. not sure if combat expert or advanced tho.

Reply:
Do we add an HJ requirement or combat (beg)?

Reply:
could do. could also make it with SJ too.